Rakefile

```diff
@@ -1,9 +1,24 @@
 require 'rake'
 require 'rake/testtask'
 
-desc 'rake with no argument will run "rake test"'
-task default: :test
+require_relative 'lib/tasks/exercise_test_tasks'
 
-Rake::TestTask.new do |task|
-  task.pattern = 'test/**/*_test.rb'
+task default: 'test'
+
+flags = ARGV.drop_while { |e| e != '--' }.drop(1)
```

Comment:
Also, in both places you use it you `join(' ')` it, so maybe you could add that back in here.